docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md

```diff
@@ -52,6 +52,8 @@ You can configure the active instance as follows:
 
 1. Click **Create**.
 
+1. If the HTTPS protocol is enabled, you can use the **Manage Peer Certificates** dialog to set up a root certificate of other instances.
```

Comment:
Make sure to do this on each standby as well.

Reply:
Added this